windows/scripts/extract_kernels.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2017, Adrian Herrera
+# All rights reserved.
+#
+# Licensed under the MIT License.
+
+"""
+Extracts the Windows kernel executable (``ntoskrnl.exe``) from an ISO. This is
```

Comment:
You must also look for ntkrnlmp and ntkrnlpa (and maybe others). Check for nt*.exe.

Reply:
Ah ok, I didn't know that. Where are these used? The README only mentions running symcheck on ntoskrnl.exe.

Reply:
The readme is incomplete.

Reply:
As someone trying to generate winmonitor_gen.c for the first time, that is not helpful. Can you please update the readme, or at least tell me here what I am supposed to do with ntkrnlmp and ntkrnlpa and I'll update it as part of this PR.

Reply:
Technically the readme is fine. Someone would take the kernel from an already running system, which will be the correct one, determined during installation. Here we are trying to future proof it by taking all the kernels that can possibly run in theory. The scenario is that we upgrade QEMU, rebuild the image, and suddenly Windows decides to use a different kernel, which would break the windows monitor.

Reply:
I agree it could be made clearer though.